modules/unsupported/ogr/ogr-jni/src/test/java/org/geotools/data/ogr/jni/JniOGRDataStoreTest.java

```diff
@@ -1,10 +1,47 @@
 package org.geotools.data.ogr.jni;
 
 import org.geotools.data.ogr.OGRDataStoreTest;
+import org.junit.Ignore;
 
 public class JniOGRDataStoreTest extends OGRDataStoreTest {
 
     public JniOGRDataStoreTest() {
         super(JniOGRDataStoreFactory.class);
     }
+
+    @Ignore
```

Comment:
Maybe it's just been a while and I'm doing it wrong, but `@Ignore` isn't working for some reason, no clue why.
Switching to just an empty `@Override`